examples/resnet-retrain/resize_images.sh

```diff
@@ -2,7 +2,7 @@
 
 FILES=$(find data -name "*.tif")
 for x in $FILES; do
-    convert $x -resize 224x224 "${x/\.tif/\.png}"
+    convert $x -resize 224x224 "${x/\.tif/.png}"
```

Comment:
This I had to change on my Mac but unsure if I'm going to break it for other OS

Reply:
Yeah, this looks suspect. Why would the one dot need to be escaped but not the other?

Reply:
not sure but my files generated ended up being renamed with a slash in it - I'll take it out of the commit. The heads up of what it should look like in the README should suffice to make sure people have the right filename

Reply:
Cool, yeah, I haven't tried this, so I can't say for sure. (: